web/admin/src/editors.js

```diff
@@ -46,19 +46,19 @@ export class SelectEditor extends PureComponent {
   }
 
   render() {
-    const {content, options, prop, size, title} = this.props
+    const {content, options, prop, size} = this.props
 
     return (
       <label className="select-editor">
-        <div className="select-editor__title">{title}</div>
         <select
-          size={size || 0}
+          // size={size || 0}
```

Comment:
why is this removed?

Reply:
The size props was not being passed to this page temporarily took it out